utils.js

```diff
@@ -3,27 +3,29 @@ const path = require('path');
 
 const glob = require('glob');
 
+function isOpenApiJson(json) {
+  return !!(json.openapi || json.swagger);
+}
+
+function isOpenApiYaml(yaml) {
+  // eslint-disable-next-line unicorn/no-unsafe-regex
+  return !!yaml.match(/\s?(openapi|swagger):\s([\s("|').0-9]+){3,}/);
```

Comment:
for case-insensitive matching:

```suggestion
  return !!yaml.match(/\s?(openapi|swagger):\s([\s("|').0-9]+){3,}/i);
```